Doc/whatsnew/3.11.rst

```diff
@@ -1407,58 +1407,125 @@ contributors are volunteers from the community.
+|                                    | | :opcode:`PUSH_NULL`                  |                    |
++------------------------------------+----------------------------------------+--------------------+
+| | :opcode:`!DUP_TOP`               | | :opcode:`COPY`                       | Stack manipulation |
+| | :opcode:`!DUP_TOP_TWO`           | | :opcode:`SWAP`                       |                    |
+| | :opcode:`!ROT_TWO`               |                                        |                    |
+| | :opcode:`!ROT_THREE`             |                                        |                    |
+| | :opcode:`!ROT_FOUR`              |                                        |                    |
+| | :opcode:`!ROT_N`                 |                                        |                    |
++------------------------------------+----------------------------------------+--------------------+
+| :opcode:`!JUMP_ABSOLUTE`           | :opcode:`JUMP_BACKWARD`                | Now relative       |
++------------------------------------+----------------------------------------+--------------------+
+| | :opcode:`!JUMP_IF_NOT_EXC_MATCH` | | :opcode:`CHECK_EXC_MATCH`            | Now performs check |
+| | :opcode:`!JUMP_IF_NOT_EG_MATCH`  | | :opcode:`CHECK_EG_MATCH`             | but doesn't jump   |
++------------------------------------+----------------------------------------+--------------------+
+| | :opcode:`!POP_JUMP_IF_FALSE`     | | :opcode:`POP_JUMP_BACKWARD_IF_FALSE` | Now relative       |
+| |                                  | | :opcode:`POP_JUMP_FORWARD_IF_FALSE`  |                    |
+| | :opcode:`!POP_JUMP_IF_TRUE`      | | :opcode:`POP_JUMP_BACKWARD_IF_TRUE`  |                    |
+| |                                  | | :opcode:`POP_JUMP_FORWARD_IF_TRUE`   |                    |
++------------------------------------+----------------------------------------+--------------------+
+| | :opcode:`!SETUP_WITH`            | :opcode:`BEFORE_WITH`                  | :keyword:`with`    |
+| | :opcode:`!SETUP_ASYNC_WITH`      |                                        | block setup        |
++------------------------------------+----------------------------------------+--------------------+
+
+.. [#opcode-binary] Replaced all numeric :opcode:`!BINARY_*`
+   and :opcode:`!INPLACE_*` instructions
+   with a single :opcode:`BINARY_OP` implementation.
+
+.. [#opcode-call] Decouples the argument shifting for methods
+   from the handling of keyword arguments
+   and allows better specialization of calls.
+
+
+.. _whatsnew311-removed-opcodes:
+
+Removed Opcodes
+---------------
+
+* :opcode:`!COPY_DICT_WITHOUT_KEYS`
+* :opcode:`!GEN_START`
+* :opcode:`!POP_BLOCK`
+* :opcode:`!SETUP_FINALLY`
+* :opcode:`!YIELD_FROM`
+
+
+.. _whatsnew311-changed-opcodes:
+
+Changed Opcodes
+---------------
+
+* Changed :opcode:`JUMP_IF_TRUE_OR_POP` and :opcode:`JUMP_IF_FALSE_OR_POP`
+  to now be relative rather than absolute.
```

Comment:
I left the Changes section as it was for now (aside from minor technical edits), but there were a handful of additional Opcode changes of roughly comparable magnitude to those mentioned that weren't noted here, mostly grouped by a few specific common alterations, that we could include in a final short table (with columns `Opcode(s)` and `Change`), if you feel its worth including.

Here are the additional changes that could be mentioned:

<details>

<summary>List of additional changes</summary>

Added where oparg

* GET_AWAITABLE

Exception representation on the stack now consist of one, not three, items

* END_ASYNC_FOR
* POP_EXCEPT
* RERAISE
* WITH_EXCEPT_START

The __exit__ function is in position 4 of the stack rather than 7.

* WITH_EXCEPT_START

If the low bit of namei is set, then a NULL is pushed to the stack before the global variable.

* LOAD_GLOBAL

i is no longer offset by the length of co_varnames

* LOAD_CLOSURE
* LOAD_DEREF
* LOAD_CLASSDEREF
* STORE_DEREF
* DELETE_DEREF

</details>